src/Symfony/Component/Console/Command/Command.php

```diff
@@ -516,7 +516,7 @@ public function getProcessedHelp()
     /**
      * Sets the aliases for the command.
      *
-     * @param string[] $aliases An array of aliases for the command
+     * @param string[]|\Traversable $aliases An array of aliases for the command
```

Comment:
@xabbuh Based on discussion on previous PR: https://github.com/symfony/symfony/pull/16624#discussion_r45559942
If still looks wrong, could you elaborate?